apps/src/userHeaderStatsigReporter/userHeaderStatsigReporter.js

```diff
@@ -1,6 +1,18 @@
 import statsigReporter from '../lib/util/StatsigReporter';
 
-document.addEventListener('DOMContentLoaded', function () {
+if (document.readyState !== 'loading') {
+  console.log('Document is ready');
+  runStatsigReporter();
+} else {
+  document.addEventListener('DOMContentLoaded', function () {
+    console.log('Document is loading');
+    console.log(`Document ready state: ${document.readyState}`);
+    runStatsigReporter();
+  });
+}
+
+function runStatsigReporter() {
+  console.log('Running Statsig Reporter');
```

Comment:
This should appear immediately after one of the above newly-added `console.log` calls, so should be interesting to confirm, but again, doesn't seem crucial.